authors/apps/authentication/tests/test_email_verification.py

```diff
@@ -76,3 +110,120 @@ def test_verification_with_bad_request(self):
             self.wrong_details,
             format='json')
         self.assertTrue(response.status_code == status.HTTP_400_BAD_REQUEST)
+
+    def test_email_send_link_verification_with_error_email(self):
+        """
+        Test wether response code will be 400
+        if request don't have email field
+        """
+        response = self.client.post(
+            self.forget_url,
+            self.user_email_with_error,
+            format='json')
+
+        self.assertIn(
+            'This field is required', str(
+                response.data))
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+
+    def test_email_send_link_verification_user_not_found(self):
+        """
+        Test wether response code will be 400
+        if user not found
+        """
+        response = self.client.post(
+            self.forget_url,
+            self.unsaved_email,
+            format='json')
+
+        self.assertIn(
+            "A user with this email was not found.", str(
+                response.data))
+        self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
+
+    def test_email_send_link_verification(self):
+        """
+        Test wether response code will be 200
+        if request has is valid
+        """
+        response = self.client.post(
+            self.forget_url,
+            self.user_email,
+            format='json'
+        )
+        self.assertIn(
+            "Confirm your email to continue", str(
+                response.data))
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+
+    def test_password_reset_with_invalid_format(self):
+        """
+        test wether the response will be 400
+        if the request has invalid format
+        """
+        user = User.objects.get(email=self.email)
+        token = default_token_generator.make_token(user)
+        hashed_email = jwt.encode(
+            {"email": self.email}, SECRET_KEY, algorithm='HS256')
+        url = 'http://localhost:8000/api/users/reset' + \
```

Comment:
Since you use it 4 times, `http://localhost:8000/api/users/reset` could be a global variable or initialize it in setUp